package.json

```diff
@@ -27,9 +27,9 @@
     "@sentry/node": "^5.9.0"
   },
   "devDependencies": {
-    "jest": "^23.2.0",
+    "jest": "^24.9.0",
     "smee-client": "^1.0.1",
-    "standard": "^11.0.1",
+    "standard": "^14.3.1",
```

Comment:
Then, after rebasing from commit 4f9de482a6ffe8aa210e75f8f2096a4d3f387177, Dependabot insert this one.